docs/website/docs/dlt-ecosystem/destinations/snowflake.md

```diff
@@ -39,11 +39,18 @@ username = "loader"
 host = "kgiotue-wn98412"
 warehouse = "COMPUTE_WH"
 role = "DLT_LOADER_ROLE"
+application = "dltHub_dlt"
```

Comment:
I don't think we should put it here, because for most users it's not relevant and it just will be a default value.

Reply:
Got it I will remove this :)